libs/sdk-embedding/src/iframe/kd.ts

```diff
@@ -1295,6 +1301,22 @@ export namespace EmbeddedKpiDashboard {
         ISetFilterParentsFailedDataBody
     >;
 
+    /**
+     * Type that represents `InsightSaved` data.
+     *
+     * @public
+     */
+    export interface IInsightSavedBody {
```

Comment:
We should keep this almost 1:1 to the AD event (where is also `visualizationObject`) for backward compability reasons.